package.json

```diff
@@ -71,6 +71,46 @@
     ],
     "yarn.lock": "yarn deduplicate-packages --list --fail"
   },
+  "eslintConfig": {
+    "extends": [
```

Comment:
Do we not need the base config `@qlik/eslint-config`?

Also what about `@qlik/eslint-config/playwright`?

Reply:
All packages also include the default @qlik/eslint-config. Should probably remove it from other repos instead. 
https://github.com/qlik-oss/dev-tools-js/blob/main/packages/eslint-config/react.js

Not so sure about playwright. I can check what happens when added